packages/example-getting-started/test/acceptance/application.test.ts

```diff
@@ -12,12 +12,12 @@ describe('Application', () => {
   let todoRepo: TodoRepository;
 
   before(givenAnApplication);
-  before(givenARestServer);
-  before(givenTodoRepository);
   before(async () => {
     await app.boot();
     await app.start();
   });
+  before(givenARestServer);
+  before(givenTodoRepository);
```

Comment:
I feel the rest server and `Todo` repository should be added before `app.boot()` and `app.start()`

Reply:
`givenTodoRepository` cannot go before `app.boot` because otherwise the repository instance won't be bound to the Application. These can go before `app.start()` but logically I don't think it makes a difference since we're not modifying the `server` instance that we get, just using that to create a `client` to test against.

Reply:
oh I misunderstood what `given*()` does, thought they create `*` for an app, but actually they get `*` from the app.